src/api/schema.prisma

```diff
@@ -184,20 +184,40 @@ model JointlyInsuredParty {
 }
 
 model ExportContract {
-  id                              String         @id @default(cuid())
-  application                     Application?   @relation("ExportContract_application", fields: [applicationId], references: [id])
-  applicationId                   String?        @map("application")
-  privateMarket                   PrivateMarket? @relation("ExportContract_privateMarket", fields: [privateMarketId], references: [id])
-  privateMarketId                 String?        @unique @map("privateMarket")
+  id                              String               @id @default(cuid())
```

Comment:
Note for PR reviewers - this file is automatically generated by KeystoneJS (from updating schema.ts)